### ISSUE TITLE
Error in validityMethod(as(object, superClass)) :    object 'Csparse_validate' not found

### ISSUE THREAD

**Issue by Aggrephagy** [xuranw/MuSiC#123]
![图片](https://github.com/xuranw/MuSiC/assets/138894823/9d0c971d-0ffa-4159-b947-dd1da91f5777)

**Comment by Aggrephagy** [comment]
The MuSiC require Matix>1.6, however , I try this pipeline error in my own data which seem like an error from Matrix.

**Comment by Jason-orthopedics** [comment]
> The MuSiC require Matix>1.6, however , I try this pipeline error in my own data which seem like an error from Matrix.

Hi, did you solve this problem?

**Comment by Aggrephagy** [comment]
No, I have not to solve this problem. However, I try sucees by the BayeisPrism, another algorithm to perform deconvolution.



---- Replied Message ----
| From | ***@***.***> |
| Date | 09/18/2023 21:38 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [xuranw/MuSiC] Error in validityMethod(as(object, superClass)) : object 'Csparse_validate' not found (Issue #123) |

The MuSiC require Matix>1.6, however , I try this pipeline error in my own data which seem like an error from Matrix.

Hi, did you solve this problem?
